web/satellite/src/utils/plugins/segment.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export class Segmentio {
+    public page(eventName: string) {
+        const segmentio = window['analytics'];
```

Comment:
this code duplicates twice in this PR.  https://github.com/storj/storj/pull/3405/files#diff-9d0cc59a386b82f910786a7e0d4f5424R49

could it be in one place?

Reply:
The main reason for this is to not have to repeat `window['analytics']`. The variable itself is in the scope of each method. We can't be sure that the property is added to window until the method is called